tools/topology/topology2/cavs-sdw-src-gain-mixin.conf

```diff
@@ -128,21 +127,39 @@ Object.Pipeline {
 			}
 		}
 	]
+}
 
-	io-gateway-capture [
+# Capture pipeline widgets
+Object.Widget {
+	alh-copier [
 		{
+			stream_name	$SDW_CAPTURE_PCM
+			dai_index	0
```

Comment:
do we need the dai_index and what would it mean here?

Reply:
same thing for all non-aggregated ALH DAIs the dai_index must be set to 0

Reply:
can it instead be omitted and default to zero, and be added only when it's useful?

It's not really a dai_index as much as a copier instance, is it?

Reply:
yes, done!